pkg/occlient/occlient.go

```diff
@@ -328,15 +328,15 @@ func (c *Client) GetCurrentProjectName() (string, error) {
 	return strings.TrimSpace(string(output)), nil
 }
 
-func (c *Client) GetProjects() (string, error) {
```

Comment:
this PR was originaly based on https://github.com/redhat-developer/ocdev/pull/250
But now I this is back to using `oc` binary :-(

Reply:
@kadel Fixed